var/spack/repos/builtin/packages/embree/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Embree(CMakePackage):
+    """High-performance ray tracing kernel"""
+
+    homepage = "https://embree.github.io/"
+    url = "https://github.com/embree/embree/archive/v3.5.1.tar.gz"
+    generator = 'Ninja'
+
+    version('3.5.2', sha256='633c059ad8156fe14552bc1e63826547bdbd9ee3572f03fb28a5fc5ebb114fcc')
```

Comment:
```suggestion
    version('3.5.2', sha256='245af8820a820af94679fa1d43a05a9c825451be0d603b6165229556adf49517')
```

Reply:
Where's that SHA coming from? It doesn't match any of the first 7 SHAs output by `spack checksum embree`.

Reply:
if its not matching with `spack checksum`, is that coming from spack mirror @matz-e ? have seen Intel releases where they change the tarballs (silently) without changing the version.

Reply:
it's an intel package… no idea why this changed.